fed/_private/grpc_options.py

```diff
@@ -89,4 +89,5 @@ def get_grpc_options(
                 }
             ),
         ),
+        ('grpc.so_reuseport', 0),
```

Comment:
Do we need a unit test to cover it?

Reply:
Yes, has added a UT, thx for the advice 🥳